raven/src/main/java/com/getsentry/raven/marshaller/json/StackTraceInterfaceBinding.java

```diff
@@ -35,21 +35,21 @@ private void writeFrame(JsonGenerator generator, StackTraceElement stackTraceEle
         generator.writeStartObject();
         generator.writeStringField(FILENAME_PARAMETER, stackTraceElement.getFileName());
         generator.writeStringField(MODULE_PARAMETER, stackTraceElement.getClassName());
-        generator.writeBooleanField(IN_APP_PARAMETER, !(removeCommonFramesWithEnclosing && commonWithEnclosing)
-            && isFrameInApp(stackTraceElement));
+        boolean inApp = !(removeCommonFramesWithEnclosing && commonWithEnclosing) && isFrameInApp(stackTraceElement);
```

Comment:
Mmmm, I didn't mean to leave this. I breakout named variables for using the debugger. They're "free" after JIT (not that I'd even care if they weren't...) so *shrug*?